lib/sourcemap.js

```diff
@@ -90,19 +93,27 @@ function SourceMap(options) {
             orig_col = info.column;
             name = info.name || name;
         }
-        generator.addMapping({
-            generated : { line: gen_line + options.dest_line_diff, column: gen_col },
-            original  : { line: orig_line + options.orig_line_diff, column: orig_col },
+        maybeAddMapping(generator, {
```

Comment:
`maybeAddMapping` produces a cleaner sourcemap by discarding mappings that add no value. Eg, if two consecutive segments point to the same source location, that provides no new information.

This is a change from the `source-map`'s `addMapping`. If you'd like, I can switch to `gen-mapping`'s `addMapping`, which gives us the same keep-everything behavior.

Reply:
Sounds like less RAM usage to me, so this is a nice change!